src/utils/hooks/useReferenceDocs.ts

```diff
@@ -38,7 +38,7 @@ export const useReferenceDocs = (currentUser: User | null) => {
 }
 
 const shouldSeeInternalDocs = (user: User): boolean => {
-  if (!user?.isAdmin) return true
+  if (!user?.isAdmin) return false
```

Comment:
Should be ` if (user?.isAdmin) return true`

What you've got there will prevent all non-admin internal users from viewing them

Reply:
The check is `!` so if NOT admin. I don't like using the not check because it takes longer to see what's the actual check, but I tried not to change this code so much....